### PR TITLE
Removed WebDriverMode

### LINKED ISSUE #190: Remove WebDriverMode.remote/local
Do we need the property or configuration option for `WebDriverMode`?

`tt.webdriver.mode=remote` requires `tt.selenium.server.url`  to be defined. That means, the information if a remote or local session should be used is kind of redundant and maybe confusing.

My proposal is, to remove the property and just decide if a remote or local WebDriver session should be created based on the presense of a Selenium server URL.

`tt.selenium.server.url=` -> `local`
`tt.selenium.server.url=http://localhost:3124` -> `remote`

### PR DESCRIPTION
**Pull request by mreiche**
# Description

Remove support of `WebDriverMode`.

Fixes #190

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/src/docs/properties/webdriver-props.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ The following types of browsrs are supported:
 a| Only defines the browser type, will be overwritten by {browser_setting}.
 | {browser_version} | na. | Only defines the browser version, will be overwritten by {browser_setting}.
 | {baseurl} | na. | URL of the first site called in a new browser session.
-| {webdriver_mode} | remote | Sets the webdriver mode. remote uses an external Selenium server
+| {webdriver_mode} | remote | Sets the webdriver mode. remote uses an external Selenium server (deprecated).
```
Comment (martingrossmann) [inline docs/src/docs/properties/webdriver-props.adoc:25]: You can also mark `tt.selenium.server.host` and `tt.selenium.server.port` as deprecated.

**Comment by martingrossmann** [comment]
In https://docs.testerra.io/testerra/stable/index.html#_setup_selenium there is needed to to describe how to add the selenium URL to `test.properties` and why it has to be done.

File: docs/src/docs/webdrivermanager/webdrivermanager-sessions.adoc [hunk 2]
```diff
@@ -34,7 +31,6 @@ However, Testerra supports using local web drivers.
 .system.properties
 [source,properties,subs="attributes"]
 ----
-{webdriver_mode}=local
```
Comment (martingrossmann) [inline docs/src/docs/webdrivermanager/webdrivermanager-sessions.adoc:37]: Here should be mentioned that you have to remove the selenium URL if exists.

Comment (martingrossmann) [reply docs/src/docs/webdrivermanager/webdrivermanager-sessions.adoc:37]: I think this would helpful..

**Review by martingrossmann — DISMISSED** [review]

**Review by gogro — APPROVED** [review]

**Review by martingrossmann — APPROVED** [review]

**Pull request merged**